test/test_fake_tensor.py

```diff
@@ -785,6 +785,14 @@ def test_aten_slice_scatter_multi_device(self):
         self.checkType(r3, "cpu", (4, 4))
         self.checkType(out, "cpu", (4, 4))
 
+    def test__adaptive_avg_pool3d_backward(self):
+        with FakeTensorMode():
+            grad_out = torch.rand(2, 3, 4, 4, 4)
+            inp = torch.rand(2, 3, 4, 4, 4).to(memory_format=torch.channels_last_3d)
+            grad_in = torch.ops.aten._adaptive_avg_pool3d_backward(grad_out, inp)
```

Comment:
This is running under fake tensor mode which uses the meta-registration from `torch/_meta_registrations.py` rather than the concrete implementation. So this test isn't hitting your code at all.

Reply:
Damn. I modified the aten library though. Do you have any idea why the registration doesn't take into account my aten modifications ?

Reply:
The meta function doesn't call into aten, you need to update it directly:
https://github.com/pytorch/pytorch/blob/499ead96ffeab66dd7b394b0986be49ed1a91172/torch/_meta_registrations.py#L2830-L2850